src/main/scala/diplomaticobjectmodel/logicaltree/LogicalTreeNode.scala

```diff
@@ -23,7 +29,7 @@ object LogicalModuleTree {
 
   def root: LogicalTreeNode = {
     val roots = tree.collect { case (k, _) if !tree.exists(_._2.contains(k)) => k }
-    assert(roots.size == 1, "Logical Tree contains more than one root.")
+    assert(roots.size <= 2, "Logical Tree contains more than two roots.")
```

Comment:
"two or more"?

Reply:
The structure of the tree is as follows: @terpstra @richardxia 

                      root
                       /   \
                     /       \
TestHarness        CoreComplex

Reply:
I don't really have much context for what is happening right now, but if we are already going from 1 to 2 roots, is there any downside to just allowing an unbounded number of roots? We essentially already have a forest.